tests/test_erosion.py

```diff
@@ -50,8 +45,8 @@ def test_validate_inputs(caplog):
                  in s.message for s in info)
 
     # Test Q_s boundaries
-    kwargs = {'Q_s': 1, 'v_m': 1, 'D': .1}
-    for illegal_input in [-1, 555]:
+    kwargs = {'Q_s': 1, 'rho_p': 1, 'v_m': 1, 'D': .1}
```

Comment:
Why have you inserted `rho_p`? It is added in line 33/34

Reply:
Did not notice. Will remove it then :)